build-support/docker/centos6/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# Use centos6 for compatibility with glibc 2.12.
+FROM centos:6
+
+# Ensure Pants runs under the 2.7.13 interpreter.
+ENV PANTS_PYTHON_SETUP_INTERPRETER_CONSTRAINTS="['CPython==2.7.13']"
+
+# TODO: Not https? Now we're really doing some science!
+RUN curl -o /etc/yum.repos.d/slc6-devtoolset.repo http://linuxsoft.cern.ch/cern/devtoolset/slc6-devtoolset.repo
+RUN rpm --import http://linuxsoft.cern.ch/cern/slc6X/x86_64/RPM-GPG-KEY-cern
```

Comment:
sketchy.. but not actually required, afaict? I'm able to yum install all of the below without this.